src/components/learner-credit-management/BudgetDetailPageOverviewAvailability.jsx

```diff
@@ -95,6 +95,46 @@ const BudgetActions = ({
     );
   }
 
+  if (status === BUDGET_STATUSES.retired) {
+    return (
+      <div className="h-100 d-flex align-items-center pt-4 pt-lg-0">
+        <div>
+          <h3>
+            <FormattedMessage
+              id="lcm.budget.detail.page.overview.budget.actions.keep.people.learning.active.budget"
+              defaultMessage="Keep people learning with an active budget"
+              description="Title for the budget actions section on the budget detail page overview"
+            />
+          </h3>
+          <p>
+            <FormattedMessage
+              id="lcm.budget.detail.page.overview.budget.actions.budget.retired"
+              defaultMessage="This budget is retired. To create a new budget, please contact your representative."
```

Comment:
Per the [Figma](https://www.figma.com/design/857ACKyCYW9XfwTf8IvznE/%22Retired%22-state-of-budgets-(ENT---8207)?node-id=223-7353&m=dev), I believe this messaging and subsequent CTA is intended to be conditional based on whether there are any active budgets or not?

<img width="253" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/bfc8e523-8f7c-4968-9f56-9186923eef22">

vs. 

<img width="225" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/12483d18-7594-4073-b22d-fa7392999de9">

"Active" budget here likely refers to any budget is not retired or expired (i.e., current or scheduled).